core/jobs/io/firebase_io.py

```
@@ -0,0 +1,217 @@
+        """
+        # `import_users()` DOES NOT accept a raw password field, so we call
+        # `create_user()`, which DOES accept one, in a loop here instead.
+        for record in batch:
+            assert record.email, 'Email is required within EMULATOR_MODE.'
+            firebase_auth.create_user(
+                uid=record.uid,
+                email=record.email,
+                disabled=record.disabled,
+                # HINT: `md5(email)` makes this consistent with the frontend.
+                # See: core/templates/services/auth.service.ts.
+                password=hashlib.md5(record.email.encode()).hexdigest(),
```

Ruff (0.15.1)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)